drivers/hmis/lib/hmis_data_cleanup/fix_enrollment_dates_20240416.rb

```diff
@@ -0,0 +1,93 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class FixEnrollmentDates20240416
+  # For Enrollments in any project EXCEPT special treatment project, where the Entry Date and Exit Date are the same:
+  # - Add one day to the exit date
+  # For Enrollments in special treatment project, where the Exit Date is before the Entry Date (or they are the same):
+  # - If exit date is in 2020, change it to 2023 but same month and day.
+  # For Bed Night Services (recordType:200) in special treatment project, where the DateProvided is in 2020
+  # - Change DateProvided to 2023 (same month and day). The project was not open in 2020
+
+  attr_accessor :special_treatment_project_id
+  def initialize(special_treatment_project_id:)
+    self.special_treatment_project_id = special_treatment_project_id
+  end
+
+  def e_t = Hmis::Hud::Enrollment.arel_table
+  def ex_t = Hmis::Hud::Exit.arel_table
+  def sv_t  = Hmis::Hud::Service.arel_table
+  def p_t = Hmis::Hud::Project.arel_table
```

Comment:
`include ArelHelper` or `include HmisArelHelper` should get you all of these

Reply:
The only difference will be sv_t=>s_t

Reply:
Oh good!

I noticed the `e_t` defined in `ArelHelper` is `GrdaWarehouse::Hud::Enrollment.arel_table`, not `Hmis::Hud::Enrollment.arel_table` (the definition that I removed from this file after adding that include statement). Does it still work because these two models use the same underlying table?

Reply:
Yep exactly, it will behave the same way